src/assets/AddIcon.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const AddIcon: React.FC = () => (
```

Comment:
To not have a component inside assets folder please just create .svg file and put the content there. After you can just import svg where you would need it